foundation.e.easy-installer.yml

```
@@ -0,0 +1,30 @@
+app-id: foundation.e.easy-installer
+runtime: org.freedesktop.Platform
+runtime-version: 21.08
+sdk: org.freedesktop.Sdk
+command: easy-installer
+
+finish-args:
+  - "--env=SNAP_NAME=easy-installer"
+  - "--env=SNAP_USER_COMMON=/var/tmp"
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
```

if it actually supports wayland

```suggestion
  - "--socket=fallback-x11"
```